README.md

```diff
@@ -40,32 +40,54 @@ It's worth noting that the attributes are an extension of [cliclopts][cliclopts]
       ```javascript
       import {cycle, vote} from '@learnersguild/game-cli'
 
-      let usage = vote.usage()
-      // <the full usage text for the vote command>
-      // - when usage is called with no arguments, it returns the full usage text
-      let args = vote.parse(['44', '45'])
-      // { _: [ '44', '45' ], help: false, h: false }
-      // the '_' attribute contains positional parameters
-      usage = vote.usage(args)
-      // undefined - we've parsed the users arguments and they didn't ask for help
-
-      args = vote.parse(['-h'])
-      // { _: [], help: true, h: true }
-      usage = vote.usage(args)
-      // <the full usage text for the vote command>
-      // - we've parsed the user's arguments and they did ask for help
-
-      args = cycle.parse(['launch', '--help'])
-      // { _: [ 'launch', '--help' ],
-      //   help: false,
-      //   h: false,
-      //   subcommand: { launch: { _: [], help: true, h: true } } }
-      // - parse() can differentiate between options for the parent and sub commands      
-      usage = cycle.usage(args)
-      // <the full usage text for the 'cycle launch' sub command>
-      // - because the '--help' option came _after_ the subcommand
+      const options = {
+        lgJWT: '<LG user SSO JWT token>',
+        lgUser: {
+          // ... LG user attributes
+        },
+        lgPlayer: {
+          // ... LG player attributes
+        }
+      }
+
+      vote.invoke(['44', '45'], console.log, options)
       ```
 
+### The Command Runner
+
+There's a built-in command-runner that can be used for development / testing. It is implemented in [/src/runner.js](/src/runner.js), and can be invoked as an npm script:
+
+        $ npm run command -- vote 44 45
+
+The command runner expects a `~/.lgrc` file to be present with attributes that will be passed to the invoke command. Without such a file, the command runner will not work. For example:
+
+```json
+{
+  "lgJWT" : "<LONG SSO JWT TOKEN>",
+  "lgUser" : {
+    "id" : "00000000-1111-2222-3333-444444444444",
+    "name" : "John Doe",
+    "handle" : "johndoe2016",
+    "email" : "johndoe2016@gmail.com",
+    "timezone" : "America/Los_Angeles",
+    "phone" : 5103456789,
+    "roles" : [
+       "player"
+    ]
+  },
+  "lgPlayer" : {
+    "id" : "00000000-1111-2222-3333-444444444444",
+    "chapter" : {
+       "name" : "Chapter Name",
+       "channelName" : "chapter-name",
+       "timezone" : "America/Los_Angeles",
+       "goalRepositoryURL" : "https://github.com/GuildCraftsTesting/web-development-js-testing"
+    }
+  }
+}
+```
```

Comment:
Not sure if this is worth adding to the README but here's a one-liner that will more or less print out what you need for your `~/.lgrc`

```bash
mongo localhost:3001/meteor --quiet --eval='printjson(db.users.aggregate([{$match: {username: "bundacia"}}, {$project : {_id: 0, lgJWT: "$services.lgSSO.lgJWT", lgUser: "$services.lgSSO.lgUser", lgPlayer: "$services.lgSSO.lgPlayer"}}]).pretty().shellPrint())'
```